src/main/java/mekanism/common/registries/MekanismRobitSkins.java

```diff
@@ -27,4 +27,5 @@ private MekanismRobitSkins() {
         }
         return Collections.unmodifiableMap(internal);
     });
+    public static final RobitSkinRegistryObject<RobitSkin> ALLAY = ROBIT_SKINS.register("allay", AllayRobitSkin::new);
```

Comment:
This can just go after the base line for readability. Also it looks like you are missing the mekanism lang provider entries for this?

Reply:
Maybe something along the lines of "Allay Costume" ?